taskfiles/utils.yml

```
@@ -103,3 +103,46 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # C++ LINTING UTILS
+  # ===
+
+  # Runs clang-format on C++ files in the given paths.
```

```suggestion
  # Runs clang-format on C++ files at the given paths.
```

Realized "at" sounds better.